### DISCUSSION TITLE
Livestream times

### DISCUSSION THREAD

**Discussion by ghost** [goggle/plugin.video.srfplaytv#19] [Ideas]
Hi there :)
When accessing the upcoming livestreams, it is not visible when they will take place. Is there a way to show the exact date & time when the corresponding livestream will be live?
Thank you :) 
Great app btw 💯

**Comment by goggle** [comment]
That's a great idea!
I have opened an issue here: https://github.com/goggle/script.module.srgssr/issues/19
If someone is interested in implementing this, go for it!

**Comment by goggle** [comment]
I have implemented this suggestion in https://github.com/goggle/script.module.srgssr/commit/82b794fbd2f36505fcb816a942676769411ff842.

New release (version 2.2.3) is on its way.
